### PR TITLE
Feat/add ci

### PR DESCRIPTION
**Pull request by mano-lis**

### CODE REVIEW

**Review by Raphael-Marchese — APPROVED** [review]

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [master, main]
+    # Remove the line above to run when pushing to master or main
+  pull_request:
+    branches: [master, main]
```
Comment (dunglas) [inline .github/workflows/linter.yml:16]:
```suggestion
on:
  push:
  pull_request:
    branches: [main]
```

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          fetch-depth: 0
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+        uses: super-linter/super-linter@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```
Comment (dunglas) [inline .github/workflows/linter.yml:57]:
```suggestion
```

File: .github/workflows/linter.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+#################################
```
Comment (dunglas) [inline .github/workflows/linter.yml:1]: I would remove all these useless comments.

File: README.md [hunk 4]
```diff
@@ -2,15 +2,15 @@
...
+This script identifies a commit in the template history which is the closest one to your project.
 Then it squashes all the updates into a commit which will be cherry-picked on the top of your working branch.
 Therefore you just have to resolve conflicts and work is done!
 
 ## Basic Usage
 1. Go to your project repository (we recommend to create a new branch)
 Copy template-sync.sh at the root of your project
 
-2. The only mandatory argument is the github or gitlab url of your template. 
+2. The only mandatory argument is the GitHub or gitlab URL of your template.
```
Comment (dunglas) [inline README.md:13]:
```suggestion
2. The only mandatory argument is the GitHub or GitLab URL of your template.
```

Does this work only with GitHub or GItLab or is any Git repo supported?

**Pull request merged**